core/src/main/resources/templates/buddy-invitation-deep-link-landing-page.vm

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width,minimum-scale=1.0, maximum-scale=1.0" />
+    <title>Yona</title>
+    <style>@media screen and (max-device-width:480px){body{-webkit-text-size-adjust:none}}</style>
+ 
+    <!-- implement javascript on web page that first first tries to open the deep link
+        1. if user has app installed, then they would be redirected to open the app to specified screen
+        2. if user doesn't have app installed, then their browser wouldn't recognize the URL scheme
+        and app wouldn't open since it's not installed. In 1 second (1000 milliseconds) user is redirected
+        to download app from app store.
+     -->
+    <script>
+    window.onload = function() {
+    <!-- Deep link URL for existing users with app already installed on their device -->
+        window.location = "${esc.javascript($deepLinkURL)}";
+    <!-- Download URL (MAT link) for new users to download the app -->
+        setTimeout("window.location = 'https://rink.hockeyapp.net/apps/e1e6d73a1d244306ba07e8a43a8565f4/app_versions/1';", 500);
```

Comment:
On my request to Kinnar how we know whether to load Google Play Store or iTunes, he responded this:

> That can be handle from Java script.
> Here is sample link: http://app.makemytrip.com/i2ozz-d2/5 from another application, if you click from Android device, it will open play store and if you open on iPhone, it will open iTunes. We need to implement similar way in email invitation link.

That we also need to add.